.kitchen.vagrant.yml

```diff
@@ -36,6 +36,9 @@ platforms:
 - name: centos-7
   driver_config:
     box: bento/centos-7
+- name: centos-8
+  driver_config:
+    box: bento/centos-8
```

Comment:
@yeoldegrove Maybe it's just an error on my part, but I get the message that this box does not exist.

It's also not listed on https://app.vagrantup.com/bento

But it does seem that the code for it has been merged 7 days ago.
https://github.com/chef/bento/commit/8c8c6a1423f7de56b085185f08b99cc583d6a1af
I guess it hasn't been officially build yet.

Reply:
Yep, I can verify that... Let's recheck in a few days.

Reply:
CentOS 8 has been released by bento: https://app.vagrantup.com/bento/boxes/centos-8